android/app/src/main/java/com/now/naaga/presentation/upload/UploadActivity.kt

```diff
@@ -182,6 +187,7 @@ class UploadActivity : AppCompatActivity(), AnalyticsDelegate by DefaultAnalytic
             if (isFormValid().not()) {
                 shortToast(getString(R.string.upload_error_insufficient_info_message))
             } else {
+                viewModel.setSuccessUpload(UploadStatus.PENDING)
```

Comment:
```suggestion
                viewModel.setSuccessUpload(UploadStatus.PENDING)
```
해당 상태 업데이트 로직을 `UploadViewModel` 클래스의 `postPlace` 안으로 이동시키는 것은 어떤가요?
뷰모델에서 나타내는 상태랑 뷰 모델에서 행해지는 일들이 동기화 되지 않을 수도 있을 것 같아서요!
(라벨 어케 다는지 몰라서 글로 남깁니다.. P3...)

Reply:
좋은 의견인 것 같아 반영했어요🦖